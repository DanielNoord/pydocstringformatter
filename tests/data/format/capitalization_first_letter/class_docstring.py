class MyClass:
    """a multi-line
    docstring"""

    class InnerClass:
        """
        a multi-line
        docstring"""


class MyClass:
    """a multi-line
    docstring
    """

    class InnerClass:
        """a multi-line
        docstring
        """


class MyClass:
    """
    a docstring"""

    class InnerClass:
        """
        a docstring"""


class MyClass:
    """A docstring"""

    class InnerClass:
        """A docstring"""
