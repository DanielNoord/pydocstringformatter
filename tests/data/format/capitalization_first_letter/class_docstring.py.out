class MyClass:
    """A multi-line
    docstring
    """

    class InnerClass:
        """A multi-line
        docstring
        """


class MyClass:
    """A multi-line
    docstring
    """

    class InnerClass:
        """A multi-line
        docstring
        """


class MyClass:
    """A docstring."""

    class InnerClass:
        """A docstring."""


class MyClass:
    """A docstring."""

    class InnerClass:
        """A docstring."""
